src/main/java/at/ac/tuwien/kr/alpha/common/atoms/Atom.java

```diff
@@ -27,31 +27,40 @@
  */
 package at.ac.tuwien.kr.alpha.common.atoms;
 
+import java.util.List;
```

Comment:
To be honest, I'm not completely sure how Intellij(?) normally organizes imports. Based on the changes here, package prefix hierarchy seems to be "own stuff (at.ac.*, com.*, org.*, java.*, javax.*)", to which I now changed my settings (not committing that on it's own in order to not produce more useless changes). However, I noticed that some files previously had wildcard imports, e.g. ```import java.util.*```, which is explicitly disabled in my settings since it's a bad practice- I'm keeping that as-is. (https://google.github.io/styleguide/javaguide.html#s3.3-import-statements, https://stackoverflow.com/questions/147454/why-using-a-wild-card-with-a-java-import-statement-bad)